mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -575,6 +578,9 @@ public void onDrawerOpened(View drawerView) {
                 case R.id.about:
                     openAbout();
                     return true;
+                case R.id.log:
+                    openLog();
+                    return true;
```

Comment:
Wouldn't it be a better idea to add an entry to the prefs list (below the debug toggle) instead of the drawer? IMHO it's not that important to justify a drawer item; I'll guess we'll need to point most people towards it either way.

Reply:
Makes sense, I'll change it.